addons/l10n_fi/static/description/index.html

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <h2 class="oe_slogan">Finnish Localization</h2>
+
+        <h3 class="oe_slogan">The essentials for all Finnish companies</h3>
+
+        <div class="oe_span6">
+            <p class="oe_mt32">
+            <h3>Authors</h3>
+            <ul>
+                <li>Miku Laitinen (Avoin.Systems)</li>
+                <li>Mikko Närjänen (Avoin.Systems)</li>
+                <li>Oskars Zālītis (Avoin.Systems)</li>
+                <li>Daniels Andersons (Avoin.Systems)</li>
+                <li>Tatiana Deribina (Avoin.Systems)</li>
+                <li>Jarmo Kortetjärvi (Oy Tawasta Technologies Ltd.)</li>
+                <li>Timo Talvitie (Vizucom Oy)</li>
+            </ul>
+            </p>
+        </div>
+        <div class="oe_span6">
+            <img src="finland.svg"
+                 height="200"
+                 style="border: 1px #eee solid;"/>
+        </div>
+    </div>
+</section>
+
+<section class="oe_container oe_dark">
+    <div class="oe_row">
+        <h2 class="oe_slogan">Features</h2>
+
+        <div class="oe_span6">
+            <p class="oe_mt32">
+            <h3>New Fields for Partners / Contacts:</h3>
+            <ul>
+                <li>Business ID (Y-tunnus)</li>
+                <li>eInvoicing Address (verkkolaskuosoite)</li>
+                <li>eInvoicing Operator (verkkolaskuvälittäjä)</li>
+            </ul>
+            You can also define your company's eInvoicing address and operator
+            from the Accounting Settings.
+            <br/><br/>
+            The eInvoicing operators can be managed from Accounting ->
```

Comment:
If it is not implemented, don't put it here.

Reply:
removed